community/libcap/build

```diff
@@ -2,6 +2,8 @@
 
 sed -i "s/CFLAGS :=/CFLAGS := \$(CFLAGS)/;/LDFLAGS :=/d" Make.Rules
 
+patch -p1 < use_sched_not_pthread.patch
+
```

Comment:
A simple sed call is better here. No need for an entire patch.

Reply:
```sh
sed -i 's/pthread\.h>/\0 #include <sched.h>/' libcap/psx.c
sed -i 's/pthread_yield/sched_yield/'         libcap/psx.c
```

Reply:
That's fine with me :smile: